### PR TITLE
Flexibus/#380/events view as tiles

### LINKED ISSUE #380: Events view as tiles
**This depends on #379 for tile component**
**#374 depends on this for event logic**

New Layout for "Events" Page:

Header and footer are out of scope.
If the event has a event-picture, then tile has this picture. If not, then we use 1 of 2 default pictures (they can be easily found in Flexibus channel)

In wide-screens every tile has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row.
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

- 'Deine Events' = I'm a member, my team is a member, I created the event
- 'Deine Events' is not shown when there is no data

### PR DESCRIPTION
**Pull request by jcremerius**
Closes #380.

### CODE REVIEW

**Comment by jkbe** [comment]
As discussed in the last meeting: Please move the corresponding issue to "in review" yourself on Waffle.io (or simply write "Closes #<issuenumber>" in the PR description) so that we have a correct overview of the state of tickets for Kanban! @jcremerius

File: app/models/event.rb [hunk 1]
```diff
@@ -186,11 +186,10 @@ def human_game_mode
...
-    if self.type == 'Rankinglist'
-      "#{I18n.t('events.index.registration_until')}: #{self.deadline} <br> #{I18n.t('events.index.start_date')}: #{self.startdate}"
-    else
-      "#{I18n.t('events.index.max_players')}: #{self.max_teams}"
-    end
+    registration_until = "#{I18n.t('events.index.registration_until')}: #{self.deadline}" if self.deadline.present?
+    start_date = "#{I18n.t('events.index.start_date')}: #{self.startdate}" if self.startdate.present?
+    return ("#{registration_until} <br> #{start_date}").html_safe if registration_until.present?
+    "#{I18n.t('events.index.max_players')}: #{self.max_teams}" if self.max_teams.present?
   end
```
Comment (Jeilef) [inline app/models/event.rb:193]: does this method really have to be in the Events model?

Comment (jcremerius) [reply app/models/event.rb:193]: Where would you place it?

File: app/assets/stylesheets/tile.css [hunk 2]
```diff
@@ -14,6 +14,18 @@
...
         width: 300px;
     }
+
+    .tile-title {
+        width: 300px;
+    }
+}
+
+
+@media screen and (max-width : 799px) and (min-width: 481px) {
```
Comment (jkbe) [inline app/assets/stylesheets/tile.css]: What do we have this block for? The condition selects a subset of elements of the `@media`'s condition above and the CSS values are the same. Remove?

**Review by jkbe — CHANGES_REQUESTED** [review]

**Review by jkbe — DISMISSED** [review]
LGTM but...

**Review by jkbe — APPROVED** [review]

**Pull request merged**